database_scripts/prod_data_investigation.sql

```diff
@@ -0,0 +1,98 @@
+--View a list of submitted ECP claims' eligibility criteria
+select * 
+  from public.early_career_payments_eligibilities
+ where created_at BETWEEN '2021-06-01T00:00:01' and '2021-10-29T14:33:14' -- e.g. Datetime format 2021-04-29T14:33:14
```

Comment:
@and-lucas-kelly same question as above about why date is 29th August and not 31st? Feels to me as if 31st is safer as its the end of the previous AcademicYear

Reply:
this is purely the investigation script to identify data